site/docs/v0.11.0/run-locally.md

```diff
@@ -0,0 +1,25 @@
+# Run Velero locally
```

Comment:
I'm not sure we want this in v0.11.0 - I'd consider anything prior to 1.0 "legacy" and not really requiring updates.